WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -67,7 +67,11 @@ + (NSArray *)routes
     [[FBRoute GET:@"/screenshot/:uuid"] respondWithTarget:self action:@selector(handleElementScreenshot:)],
     [[FBRoute GET:@"/wda/element/:uuid/accessible"] respondWithTarget:self action:@selector(handleGetAccessible:)],
     [[FBRoute GET:@"/wda/element/:uuid/accessibilityContainer"] respondWithTarget:self action:@selector(handleGetIsAccessibilityContainer:)],
-#if !TARGET_OS_TV
+#if TARGET_OS_TV
+    [[FBRoute POST:@"/element/:uuid/focuse"] respondWithTarget:self action:@selector(handleFocuse:)],
```

Comment:
these are not standard endpoints, so it makes sence to prefix them all with `wda` (except of `attribute/focused`)